front/components/assistant/conversation/ConversationTitle.tsx

```diff
@@ -39,16 +51,116 @@ export function ConversationTitle({
       workspaceId: owner.sId,
     });
 
+  const onTitleChange = async (title: string) => {
+    try {
+      const res = await fetch(
+        `/api/w/${owner.sId}/assistant/conversations/${conversationId}`,
+        {
+          method: "PATCH",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify({
+            title,
+            visibility: conversation?.visibility,
+          }),
+        }
+      );
+      await mutate(
+        `/api/w/${owner.sId}/assistant/conversations/${conversationId}`
```

Comment:
You want to mutate conversations as well to update the left-hand pane?

Reply:
Indeed. That's a whole lot of refetching for just a title :(

Reply:
Done